### ISSUE TITLE
2020.6 Crash after selecting preferences!

### ISSUE THREAD

**Issue by PixelPowerUSA** [smeighan/xLights#1950]
I was packaging a sequence and when I tried to open preferences to make sure all the files would be copied, (exclude presets, exclude audio section), It crashed. Tried a second time and it did the same thing. Crashes after selecting preferences but before it opens. Before I submitted this it worked fine the third time selected preferences.
[xLights_dbgrpt-1088-20200224T094432.zip](https://github.com/smeighan/xLights/files/4245141/xLights_dbgrpt-1088-20200224T094432.zip)

**Comment by kcobbva** [comment]
I am having the same issue.  Windows 10 1909.   Open Xlights 2020.6 (updated today), then open preferences.  This is repeatable.   However, if I open xlights, then open a sequence, then open preferences, I am able to get into it.    Again these steps have been repeatable.
[xLights_dbgrpt-21728-20200224T202702.zip](https://github.com/smeighan/xLights/files/4247684/xLights_dbgrpt-21728-20200224T202702.zip)

**Comment by keithsw1111** [comment]
Can you try this version ... I have seen it crash a couple of time in the release build but this one hasnt been crashing for me: https://www.dropbox.com/s/7anetj7430enf8s/xLights64_2020_06.exe?dl=0

**Comment by kcobbva** [comment]
Keith,

I tried this on my VM up in Azure this morning and it worked fine.   I’ll try it on my main system when I get home later today.   The system in Azure had the same issue as my home system, and I tried that version first again today before installing this version.    With the original 2020.06 it died touching preferences.  This version worked like a champ.

Thanks very much,

Kevin Cobb

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by jamesvh724** [comment]
I am experiencing the same problem.   I click File Preferences and xlights crashes.   Attaching my dump file as well.   
[xLights_dbgrpt-1760-20200225T140916.zip](https://github.com/smeighan/xLights/files/4251724/xLights_dbgrpt-1760-20200225T140916.zip)

**Comment by kcobbva** [comment]
Keith,

This has also cleared the issue on my laptop which is my primary workplace.   So both the aforementioned azure vm and this laptop – both windows 10, work great with this version.   Thank you,

Kevin Cobb

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

From: Kevin Cobb<mailto:kevinjcobb@live.com>
Sent: Tuesday, February 25, 2020 8:00 AM
To: smeighan/xLights<mailto:reply@reply.github.com>; smeighan/xLights<mailto:xLights@noreply.github.com>
Cc: Comment<mailto:comment@noreply.github.com>
Subject: RE: [smeighan/xLights] 2020.6 Crash after selecting preferences! (#1950)

Keith,

I tried this on my VM up in Azure this morning and it worked fine.   I’ll try it on my main system when I get home later today.   The system in Azure had the same issue as my home system, and I tried that version first again today before installing this version.    With the original 2020.06 it died touching preferences.  This version worked like a champ.

Thanks very much,

Kevin Cobb

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10
